README.md

```diff
@@ -447,6 +447,98 @@ or from the tenant resource (or from the application, in the case of account sto
   $accountStoreMapping->accountStore = $directory; // this could also be a group
   $application->createAccountStoreMapping($accountStoreMapping);
   ```
+  
+### Custom Data
+You may find it useful to store your own custom data on resources.  With the Stormpath SDK, we make it easy to do so. 
+All resources support a Custom Data collection for your own custom use.  The CustomData resource is a schema-less JSON 
+object (aka ‘map’, ‘associative array’, or ‘dictionary’) that allows you to specify whatever name/value pairs you wish.
+
+The CustomData resource is always connected to a resource and you can always reach it by calling the 
+getCustomData() method on the resource instance:
+
+```php
+  $application = \Stormpath\Resource\Application::get($applicationHref);
+
+  $applicationCustomData = $application->getCustomData();
```

Comment:
With the "Magic" functionality though, this call becomes:

```php
$applicationCustomData = $application->customData;
```

Same with the rest of the calls to the "getCustomData()" method.